README.md

```diff
@@ -5,7 +5,7 @@
 ![GitHub stars](https://img.shields.io/github/stars/opendilab/awesome-model-based-RL?color=yellow)
 ![GitHub forks](https://img.shields.io/github/forks/opendilab/awesome-model-based-RL?color=9cf)
 [![GitHub license](https://img.shields.io/github/license/opendilab/awesome-model-based-RL)](https://github.com/opendilab/awesome-model-based-RL/blob/main/LICENSE)
-<!-- ![visitors](https://visitor-badge.glitch.me/badge?page_id=opendilab/awesome-model-based-RL) -->
+![visitors](![visitor badge](https://visitor-badge.lithub.cc/badge?page_id=opendilab.awesome-model-based-RL&left_text=Visitors))
```

Comment:
There seem to be extra symbols, e.g., (). But if canceling (), it does not work properly?

Reply:
I found that there must be characters before Vistor Badge. So I put them all in one line and fix it.